generate.sh

```diff
@@ -61,6 +62,9 @@ sed -i -e "s/Merlin's/$project_name/g" web/app/static/manifest.json
 # This is about the web
 cd web || exit
 
+# Replace property values across stylesheets with their respective ui_kit_value
```

Comment:
This isn't limited to stylesheet files - I don't know if that's a problem? If so, here's one way to do that:
```bash
egrep -lR "ui_kit_value" $(find ./app -name '*.css' -o -name '*.scss') | xargs ...
```
(you might need to edit the list of possible file extensions in the `find`. `-o` means _or_)